person-service/build.gradle.kts

```diff
@@ -1,38 +1,36 @@
 plugins {
-    kotlin("jvm")
+    kotlin("jvm") version "2.0.21"
+    kotlin("kapt") version "2.0.21"
     kotlin("plugin.allopen") version "2.0.21"
+    kotlin("plugin.noarg") version "2.0.21"
     id("io.quarkus")
 }
 
-repositories {
-    mavenCentral()
-    mavenLocal()
-}
+val mapstructVersion: String by project
 
 dependencies {
+    implementation("io.quarkus:quarkus-hibernate-orm-panache-kotlin")
+    implementation("io.quarkus:quarkus-jdbc-postgresql")
+//    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
 
+    testImplementation("com.thehuginn:quarkus-sql-testing:1.0.0-SNAPSHOT")
+    implementation(kotlin("stdlib-jdk8"))
 }
 
 kotlin {
     jvmToolchain(21)
 }
 
-//java {
```

Comment:
kludne odmazni vsetky comments, ja som si nechal tie jedny commenty, nech to ptm viem nastavit, ak bude treba, ale asi to odmaznem aj ja ptm